commands/diagnostics.vegman

```diff
@@ -218,12 +250,31 @@ function cmd_intrusion_clear {
   chip=$1
   pin=$2
 
+  local state
   if [[ -z "${chip}" ]]; then
-    echo "Intrusion detection reset is not available for this product"
-    return 1
+    # Check for IRQ_BMC_PCH_SMI_LPC pin on VEGMAN-R hardware
+    set -- $(gpiofind ${GEN_SMI_GPIO_PIN}) "" ""
+    chip=$1
+    pin=$2
+    if [[ -z "${chip}" ]]; then
+      echo "Intrusion detection reset is not available for this product"
+      return 1
+    fi
+    state=$(get_intrusion_state)
+    echo "Current chassis intrusion state: $(get_intrusion_state)"
+    state=$(get_intrusion_request_status)
+    echo "Current ${state}"
+    echo -n "Request intrusion sensor state reset? "
```

Comment:
This must only be asked if no `-y|--yes` option is provided. Check how that is done below for other vegmans.